lib/inspec/base_cli.rb

```diff
@@ -409,27 +251,27 @@ def vendor_deps(path, opts)
 
     def configure_logger(o)
```

Comment:
may be we can also replace `o` with more descriptive argument name ?

Reply:
We're currently working with a skeleton crew for PR reviews, so we're being careful not to introduce changes that would drive up line counts.  This PR does not touch the line you mention, so it neither improves nor worsens the situation.  I'd rather change it as well.

We would certainly welcome a separate PR in which you make any readability and clarity changes you see fit.